src/Servant/Multipart.hs

```diff
@@ -305,7 +309,7 @@ fuzzyMultipartCTCheck ct
 --
 --   'generalOptions' lets you specify mostly multipart parsing
 --   related options, such as the maximum file size, while
---   'tmpOptions' lets you configure aspects specific to
+--   'options' lets you configure aspects specific to
```

Comment:
Shall we rename the field to `backendOptions` to make it crystal clear what `options` is?